src/QMCDrivers/DMC/DMCBatched.cpp

```diff
@@ -532,6 +532,9 @@ bool DMCBatched::run()
       run_time_manager.markStop();
       break;
     }
+    // Progress messages if not aborting
+    if (!myComm->rank())
```

Comment:
I would prefer to see this printout even the run aborts.

Reply:
Addressed